### PR TITLE
 Centos 7 CIS Benchmark Rules reduced config and version headers

### PR DESCRIPTION
**Pull request by ajegher-sn**
This pull request will add the file with the CentOS 7 benchmarks IDs and add to the other definition files the version headers. This should complete the work of PR  #11.

### CODE REVIEW

File: configs/defs/cos.textproto [hunk 1]
```diff
@@ -2913,6 +2913,7 @@ benchmark_configs: {
   compliance_note: {
     version: { cpe_uri: "cpe:/o:cos:cos_linux:93" version: "1.0.0" benchmark_document: "CIS Container-Optimized OS" }
     version: { cpe_uri: "cpe:/o:cos:cos_linux:97" version: "1.0.0" benchmark_document: "CIS Container-Optimized OS" }
+    version: { cpe_uri: "cpe:/o:centos:centos:7.*" version: "3.1.2" benchmark_document: "CIS Centos Linux 7.x" }
```
Comment (erikvarga) [inline configs/defs/cos.textproto:2916]: This looks like a generic linux check and not a COS one. Can you move it into the generic_linux.textproto?

File: configs/defs/fallback.textproto [hunk 2]
```diff
@@ -9,6 +9,7 @@ benchmark_configs: {
   id: "cramfs-mounting-disabled-fallback"
   compliance_note: {
     version: { cpe_uri: "fallback" version: "2.0.0" benchmark_document: "CIS Distribution Independent Linux" }
+    version: { cpe_uri: "cpe:/o:centos:centos:7.*" version: "3.1.2" benchmark_document: "CIS Centos Linux 7.x" }
```
Comment (erikvarga) [inline configs/defs/fallback.textproto:12]: We'd like to keep the fallback benchmarks defs separate from the rest and only use it in the fallback proto (even if that causes some duplication).

Can you use equivalents from the generic_linux benchmarks here instead? If they don't exist, feel free to copy them over from fallback.textproto into generic_linux.textproto and just remove the "-fallback" suffix from the ID.

File: configs/defs/fallback.textproto [hunk 3]
```diff
@@ -1248,7 +1255,7 @@ benchmark_configs: {
       "re-running the file display command.\n"
     cis_benchmark: {
       profile_level: 1
-      severity: LOW
+      severity: HIGH
```
Comment (erikvarga) [inline configs/defs/fallback.textproto:1258]: Why the change in severity? Let's keep this LOW for now as our current policy is to consider config related security findings (as opposed to CVEs) low severity.

File: configs/defs/fallback.textproto [hunk 4]
```diff
@@ -9285,7 +9357,7 @@ benchmark_configs: {
     remediation: "Correct or justify any items discovered by the CIS document's Audit step."
     cis_benchmark: {
       profile_level: 1
-      severity: LOW
+      severity: HIGH
```
Comment (erikvarga) [inline configs/defs/fallback.textproto:9360]: Same here, keep LOW

File: configs/defs/cos.textproto (hunk 1 above)
Comment (ajegher-sn) [reply configs/defs/cos.textproto:2916]: Do you mean the whole benchmark?

File: configs/defs/fallback.textproto (hunk 2 above)
Comment (ajegher-sn) [reply configs/defs/fallback.textproto:12]: There are around 60 checks that needs to be copied from fallback to generic, would it be ok anyway?

Comment (erikvarga) [reply configs/defs/fallback.textproto:12]: Yeah, that's fine. Just one thing to note: The fallback checks were made to work on things like binary-only containers that don't have a filesystem. Thus the checks pass if the checked file doesn't exist (e.g. see the `aide-installed-fallback` check). If any of these are copied into the generic linux checks they should not have these check alternatives as the generic linux checks are meant to be run on full-fledged linux systems.

File: configs/defs/cos.textproto (hunk 1 above)
Comment (erikvarga) [reply configs/defs/cos.textproto:2916]: Yeah, the whole benchmark. If it's used by OSes other than COS then it should be defined in the generic linux def file.

**Comment by ajegher-sn** [comment]
@erikvarga I have edited the files as requested. But I see that in 15 benchmarks defined in generic_linux, 13 used by CentOS, the image scan look for files in folder _/usr/share/google/security_. For example, benchmark "strong-mac-algorithms-used". This looks like they should be used in cos only and not in a generic linux image. Should I fix them by copying them to the cos definition and removing the _/usr/share/google/security_ part from the scan?

**Comment by erikvarga** [comment]
Hm, yeah, if the checks refer to COS-specific files then they should be under the COS checks. Can you copy them over to COS (add a "-cos" suffix to the IDs) and remove the COS specific paths from the ones left in generic_linux.textproto?

**Comment by ajegher-sn** [comment]
> Hm, yeah, if the checks refer to COS-specific files then they should be under the COS checks. Can you copy them over to COS (add a "-cos" suffix to the IDs) and remove the COS specific paths from the ones left in generic_linux.textproto?

Done!
